rust/sbp/src/messages/mod.rs

```diff
@@ -8,6 +8,11 @@
 // EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
 // WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
 //! SBP message definitions.
+
+//****************************************************************************
+// Automatically generated from yaml/
```

Comment:
really tiny thing, dont think this got generated correctly, `yaml/(((filepath)))` should maybe be `generator/sbpg/targets/resources/rust/sbp_messages_mod.rs`

Reply:
thanks for pointing this out